src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/LoggerMessageDefineAnalyzer.cs

```diff
@@ -257,6 +257,10 @@ private void AnalyzeFormatArgument(OperationAnalysisContext context, IOperation
                         {
                             text += textSyntax.Text;
                         }
+                        else if (interpolatedStringContent is IInterpolationOperation { ConstantValue: { HasValue: true, Value: string constantValue } })
+                        {
+                            text += constantValue; // works for e.g. nameof
```

Comment:
expected to work for nameof, does not seem to be supported yet

Reply:
Can we tag the folks delivering that change and link the issue so that it's clear we're blocked on it?